.changelog/5028b91c-7174-4v65-9adf-c787e66cd12c.json

```
@@ -0,0 +1,9 @@
+{
+    "id": "5028b91c-7174-4v65-9adf-c787e66cd12c",
+    "type": "bugfix",
+    "collapse": true,
```

One module, nothing to collapse

```suggestion
    "collapse": false,
```